dependencyManagement/build.gradle.kts

```diff
@@ -15,7 +15,7 @@ val DEPENDENCY_BOMS = listOf(
   "com.fasterxml.jackson:jackson-bom:2.13.0",
   "com.google.guava:guava-bom:31.0.1-jre",
   "com.google.protobuf:protobuf-bom:3.18.1",
-  "com.linecorp.armeria:armeria-bom:1.11.0",
+  "com.linecorp.armeria:armeria-bom:1.13.2",
```

Comment:
Needed this bugfix https://github.com/line/armeria/commit/0018233e7a9f6eab6da51f046fc3203135aa1a55